packages/file-type-icons/typings/import-json/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.json' {
```

Comment:
You shouldn't need this typing file. Let me know what error you're getting without it and I can suggest another strategy.

Reply:
importing json files in that way is only available on typescript > 2.9.x. This repo uses 2.8.x